config/locales/en.yml

```diff
@@ -42,57 +48,47 @@ en:
         user:
           attributes:
             full_name:
-              blank: "can't be blank"
-              too_short: "is too short(minimum 3 symbols)"
-              taken: "is already taken"
-            email:
-              blank: "can't be blank"
-              bad_email_format: "should have next format: xxx@xxx.xxx"
-              taken: "is already taken"
-            password:
-              blank: "can't be blank"
-              confirmation: "passwords doesn't match"
               too_short: "is too short(minimum %{count} symbols)"
-            password_confirmation:
-              confirmation: "passwords doesn't match"
-            current_password:
-              blank: "can't be blank"
-        test_setting:
-          attributes:
-            name:
-              blank: "can't be blank"
-            time_to_pass:
-              blank: "can't be blank"
-            text:
-              blank: "can't be blank"
-        tiding:
-          attributes:
-            title:
-              blank: "can't be blank"
-              too_short: "is too short(minimum 3 symbols)"
-              taken: "is already taken"
-            text:
-              blank: "can't be blank"
-        comment:
-          attributes:
-            name:
-              blank: "can't be blank"
-            text:
-              blank: "can't be blank"
+
+  buttons:
+    back: "Back"
+    remove: "Remove"
+    edit: "Edit"
+    destroy: "Remove"
+    show: "Shwo"
+    submit: "Save"
+    save: "Save"
+    login: "Login"
+    sign_up: "Sign Up"
+
+  user:
```

Comment:
this construction is not related to any parent. What does it mean? Why just `user`?
I see here few logical constructions. There are `activerecord: models: attributes:` ; `buttons:`, `errors` and `views: some page`. 
Put this construction also under some logical part.